bundle/deploy/state.go

```diff
@@ -0,0 +1,128 @@
+package deploy
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/fileset"
+)
+
+const DeploymentStateFileName = "deployment-state.json"
```

Comment:
The path will be `~/.bundle/name/target/state/deployment-state.json`. Can we use just `deployment.json`?